cpp/CMakeLists.txt

```diff
@@ -58,6 +59,7 @@ include(CMakeDependentOption)
 cmake_dependent_option(RAFT_USE_FAISS_STATIC "Build and statically link the FAISS library for nearest neighbors search on GPU" ON RAFT_COMPILE_LIBRARIES OFF)
 
 message(VERBOSE "RAFT: Build RAFT unit-tests: ${BUILD_TESTS}")
+message(VERBOSE "RAFT: Building raft C++ benchmarks: ${BUILD_BENCH}")
```

Comment:
This option also needs to be added to `build.sh` to match cuml. Also important that we disable the benchmarks by default in `build.sh` so the artifacts doesn’t end up in our conda packages. I can see an argument for maybe doing this eventually but we will want to release a specific package just for the benchmarks.

Reply:
Hmm, it seems like the `bench` or `prims-bench` parameters in [cuml/build.sh](https://github.com/rapidsai/cuml/blob/4c1d67120cfbd18cb1e9f1b2f9d2a0395b8df8e1/build.sh) are not passed along to `CMakeLists.txt`, and I haven't found anything by `bench`/`BENCH` keywords in the `ci` of cuml.

As for your suggestion, please check my question in the discussion above.